src/stackTools/stackPrefetch.js

```diff
@@ -239,7 +239,8 @@ function prefetch(element) {
       requestType,
       preventCache,
       doneCallback,
-      failCallback
+      failCallback,
+      true
```

Comment:
While `addToBeginning` could potentially default to true, it feels outside the scope of this PR

Reply:
I'm agree, I should have made another PR for this change.

Reply:
Can I leave that or do you prefer a new PR ?